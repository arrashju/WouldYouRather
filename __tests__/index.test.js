import { render } from "@testing-library/react";
import Home from "../pages/index";
import "jest-styled-components";

describe("Renders home page", () => {
  it("home render should match snapshot", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
