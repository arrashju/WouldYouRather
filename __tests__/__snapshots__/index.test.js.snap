// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders home page home render should match snapshot 1`] = `
.c0 {
  margin-top: 75px;
  margin-bottom: 100px;
  padding: 0 5%;
}

.c3 {
  background: none;
  padding: 12px 15px;
  border: 1px solid white;
  border-radius: 8px;
  font-size: 14px;
  color: white;
  -webkit-transition: background 0.2s ease;
  transition: background 0.2s ease;
  -webkit-transition: border 0.2s ease;
  transition: border 0.2s ease;
}

.c3:hover {
  background: white;
  color: #2777C1;
  cursor: pointer;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #2777C1;
  border-radius: 10px;
  height: 300px;
  width: 90%;
  padding: 5%;
}

.c5 {
  display: grid;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  grid-template-columns: repeat(auto-fit,minmax(240px,1fr));
  grid-gap: 25px;
}

.c6 {
  border-radius: 10px;
  aspect-ratio: 1/1;
  font-size: 1em;
  overflow: hidden;
  padding: 10%;
  line-height: 125%;
  font-size: max(24px,3vw);
  color: white;
  font-weight: 500;
  -webkit-filter: contrast(105%);
  filter: contrast(105%);
  overflow: hidden;
}

.c7 {
  display: block;
  background: white;
  padding: 15px 20px;
  border: 1px solid white;
  border-radius: 8px;
  font-size: 16px;
  font-weight: 500;
  color: #222;
  margin-top: 25px;
  -webkit-transition: background 0.2s ease;
  transition: background 0.2s ease;
  -webkit-transition: border 0.2s ease;
  transition: border 0.2s ease;
}

.c7:hover {
  background: #ddd;
  cursor: pointer;
}

.c4 {
  font-size: max(24px,2.3vw);
  margin-bottom: 55px;
  font-weight: 500;
  color: Black;
}

.c2 {
  font-size: max(24px,2.5vw);
  text-align: center;
  line-height: 150%;
  font-weight: 500;
  color: white;
  text-wrap: balance;
}

.c2:not(:last-child) {
  margin-bottom: 30px;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        style="display: flex; flex-direction: column; justify-content: center; align-items: center; min-height: 110px;"
      >
        <div
          class="c2"
        >
          Would you rather forfeit an Oscar or have a lifetime ban from the Academy?
        </div>
        <button
          class="c3"
        >
          Answer
        </button>
      </div>
    </div>
  </div>
  <div
    class="c0"
  >
    <div
      class="c4"
    >
      Learn Opinions
    </div>
    <div
      class="c5"
    >
      <div
        class="c6"
        style="background-size: cover;"
      >
        Sports fanatic?
        <button
          class="c7"
        >
          View Opinions
        </button>
      </div>
      <div
        class="c6"
        style="background-size: cover;"
      >
        Business enthusiast?
        <button
          class="c7"
        >
          View Opinions
        </button>
      </div>
    </div>
  </div>
</div>
`;
